src/pal/src/include/pal/virtual.h

```diff
@@ -105,6 +105,94 @@ BOOL VIRTUALOwnedRegion( IN UINT_PTR address );
 
 #ifdef __cplusplus
 }
+
+/// <summary>
```

Comment:
As much as I love using the C# style comments, the comment style in the PAL is different and I think it would be nice to use the existing style for the consistency sake.